src/main/models/JobApplyRoleRequest.ts

```diff
@@ -0,0 +1,5 @@
+export type JobApplyRoleRequest = {
+    email: string,
```

Comment:
I assume the email is the primary key for your user table?

Reply:
Email and RoleID are composite keys to prevent a user applying to the same role twice